### ISSUE TITLE
ArticleCard: Content filtering

### ISSUE THREAD

**Issue by Pete-Robelou** [ceph/ceph.io#153]
The `ArticleCard` is used for both `blog-posts` and `press-releases` for all instances wherever they appear in the site.

I spotted a difference between the Blog listing page and the Blog search results page, they display the caption content slightly differently.

Blog listing:
<img width="328" alt="Screenshot 2021-06-09 at 12 05 07" src="https://user-images.githubusercontent.com/13508865/121343802-f879da00-c91a-11eb-9e7a-ff2ba875270c.png">

Blog search results:
<img width="315" alt="Screenshot 2021-06-09 at 12 05 53" src="https://user-images.githubusercontent.com/13508865/121343891-0e879a80-c91b-11eb-90ee-cd98795ef186.png">

Ignore the line-break character for now but if you look at the punctuation you'll notice a difference. 

Essentially what is happening here is they're being fed different content. The search results uses `search-output-json` which has been passed through a filter removing punctuation while the listing page uses unfiltered content directly from the `blog-posts` collection.

To bring this inline I think we should update `ArticleCard` itself to filter the content rather than relying on pre-filtered content. That way we know it's output will always be consistent.

**Comment by adamduncan** [comment]
+1 on removing the pilcrow from the caption content. Spotted that when reviewing the site this morn. Good spot.

**Comment by Pete-Robelou** [comment]
This was a simple fix.

I've rejigged `cleanSearchOutput` to `cleanCardContent` as it's more specific to `card` components rather than `search`, we use it for the `event` card too.

I've also removed a few characters from the punctuation that it removes so the card captions read a little better.

And finally I replaced `removeHtml` with `cleanCardContent` inside the `ArticleCard` component so it consistently outputs the same content.
